### DISCUSSION TITLE
Does optical flow work in infrared light?

### DISCUSSION THREAD

**Discussion by serkanMzlm** [bitcraze/discussions#387] [Q&A]
Will the PMW3901 sensor work under infrared light?
(I want to use this sensor in a dark environment, but I don't want to use led style lighting in this environment.)

**Comment by ataffanel** [comment]
As far as I know it should: it does not have an IR filter and I have a vague recollection of someone using an infrared LED to light-up the floor under the flow deck but I cannot find a trace of it.

However this is nothing we have tested so far so it still requires investigation. If it works we would be very interested to know about it!

**Comment by artcontr** [comment]
I'm very interested to have feedback on this because I want to do a similar thing : https://github.com/orgs/bitcraze/discussions/454
